rpm/openssl.spec

```diff
@@ -1,483 +1,246 @@
-Name: openssl
-Version: 1.1.1v
-# Do not forget to bump SHLIB_VERSION on version upgrades
-Release: 1
-
-# We have to remove certain patented algorithms from the openssl source
-# tarball with the hobble-openssl script which is included below.
-# The original openssl upstream tarball cannot be shipped in the .src.rpm.
-Source: openssl-%{version}.tar.gz
-Source1: hobble-openssl
-Source2: Makefile.certificate
-Source6: make-dummy-cert
-Source7: renew-dummy-cert
-Source9: opensslconf-new.h
-Source10: opensslconf-new-warning.h
-Source11: README.FIPS
-Source12: ec_curve.c
-Source13: ectest.c
-Source14: fixpatch
-# Build changes
-Patch1: openssl-1.1.1-build.patch
-Patch2: openssl-1.1.1-defaults.patch
-Patch3: openssl-1.1.1-no-html.patch
-Patch4: openssl-1.1.1-man-rename.patch
-# Functionality changes
-Patch31: openssl-1.1.1-conf-paths.patch
-Patch32: openssl-1.1.1-version-add-engines.patch
-Patch33: openssl-1.1.1-apps-dgst.patch
-Patch36: openssl-1.1.1-no-brainpool.patch
-Patch37: openssl-1.1.1-ec-curves.patch
-Patch38: openssl-1.1.1-no-weak-verify.patch
-Patch40: openssl-1.1.1-disable-ssl3.patch
-Patch41: openssl-1.1.1-system-cipherlist.patch
-Patch42: openssl-1.1.1-fips.patch
-Patch44: openssl-1.1.1-version-override.patch
-Patch45: openssl-1.1.1-weak-ciphers.patch
-Patch46: openssl-1.1.1-seclevel.patch
-Patch47: 0001-ts-Use-SHA256-instead-of-SHA1-and-SHA512-instead-MD5.patch
-Patch48: openssl-1.1.1-fips-post-rand.patch
-Patch49: openssl-1.1.1-evp-kdf.patch
-Patch50: openssl-1.1.1-ssh-kdf.patch
-Patch51: openssl-1.1.1-intel-cet.patch
-Patch60: openssl-1.1.1-krb5-kdf.patch
-Patch61: openssl-1.1.1-edk2-build.patch
-Patch62: openssl-1.1.1-fips-curves.patch
-Patch65: openssl-1.1.1-fips-drbg-selftest.patch
-Patch66: openssl-1.1.1-fips-dh.patch
-Patch67: openssl-1.1.1-kdf-selftest.patch
-Patch69: openssl-1.1.1-alpn-cb.patch
-Patch70: openssl-1.1.1-rewire-fips-drbg.patch
-# Backported fixes including security fixes
-Patch52: openssl-1.1.1-s390x-update.patch
-Patch53: openssl-1.1.1-fips-crng-test.patch
-Patch55: openssl-1.1.1-aes-asm-aesv8-armx.pl-20-improvement-on-ThunderX2.patch
-Patch56: openssl-1.1.1-s390x-ecc.patch
-Patch57: openssl-1_1-Optimize-RSA-armv8.patch
-Patch58: openssl-1_1-Optimize-AES-GCM-uarchs.patch
-Patch59: openssl-1_1-Optimize-AES-XTS-aarch64.patch
-
-License: OpenSSL
-URL: http://www.openssl.org/
-BuildRequires: make
-BuildRequires: gcc 
-BuildRequires: coreutils, perl, sed, zlib-devel
-BuildRequires: lksctp-tools-devel
-# /usr/bin/cmp
-BuildRequires: diffutils
-# /usr/bin/rename
-BuildRequires: util-linux
-
-
-Requires: coreutils, make
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}
+%bcond_with checks
+
+%define ssletcdir %{_sysconfdir}/pki/tls
+%define sover 3
+%define _rname openssl
+%define man_suffix 3ssl
+Name:           openssl
+Version:        3.2.3
+Release:        0
+Summary:        Secure Sockets and Transport Layer Security
+License:        Apache-2.0
```

Comment:
ASL 2.0 by the current form we use.

Reply:
We shouldn't drop the SPDX tag.. Doing so only makes the indexing of licenses harder.

Reply:
We've had this conversation already.